src/frontend/src/lib/components/hero/Hero.svelte

```diff
@@ -16,5 +16,6 @@
 	<div class="py-4">
 		<Alpha />
 	</div>
+
```

Comment:
Format?

Reply:
yep! but strangely it was my IDE and not prettier... but for prettier (and the CIs) it was acceptable. I will have to take a look into it at some point